app/models/authentication.rb

```diff
@@ -2,6 +2,7 @@ class Authentication < ApplicationRecord
   include PasswordConcern
   include TenancyConcern
   include EventConcern
+  include AvailabilityStatusConcern
```

Comment:
Authentication doesn't have availability status, can be removed

Reply:
Yes it does. You can find it in the database scheme bellow. I am not sure, if we are using these fields, since I have them empty in all of my data. If we don't, we should remove these DB columns. What do you thin about that @slemrmartin and @lindgrenj6 ?
![Screenshot from 2020-12-18 12-16-21](https://user-images.githubusercontent.com/19405716/102614186-16b50900-4134-11eb-9ea0-6cc9423c0e95.png)

Reply:
Theoretically it can be used in the future so we don't need to remove it now, thus we should keep this concern for this model